<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkPage</class>
 <widget class="QWidget" name="NetworkPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>563</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Network&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9_3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Download directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="downloadDirEdit">
       <property name="toolTip">
        <string>Enter the download directory (leave empty to use the default location)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadDirButton">
       <property name="toolTip">
        <string>Select the download directory via a directory selection dialog</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="requestFilenameCheckBox">
     <property name="toolTip">
      <string>Select to ask the user for a download filename</string>
     </property>
     <property name="text">
      <string>Request name of downloaded file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="proxyGroup">
     <property name="toolTip">
      <string>Select to use a web proxy</string>
     </property>
     <property name="title">
      <string>Use network proxy</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="systemProxyButton">
        <property name="toolTip">
         <string>Select to use the system proxy configuration</string>
        </property>
        <property name="text">
         <string>Use system proxy configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="manualProxyButton">
        <property name="toolTip">
         <string>Select to use an application specific proxy configuration</string>
        </property>
        <property name="text">
         <string>Manual proxy configuration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Manual proxy settings</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Proxy-Type:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="proxyTypeCombo">
           <property name="toolTip">
            <string>Select the type of the proxy</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Proxy-Host:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="proxyHostEdit">
           <property name="toolTip">
            <string>Enter the name of the proxy host</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Proxy-Port:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="proxyPortSpin">
           <property name="toolTip">
            <string>Enter the proxy port</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="proxyUserEdit">
           <property name="toolTip">
            <string>Enter the username for the proxy</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="proxyPasswordEdit">
           <property name="toolTip">
            <string>Enter the password for the proxy</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>571</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>downloadDirEdit</tabstop>
  <tabstop>downloadDirButton</tabstop>
  <tabstop>requestFilenameCheckBox</tabstop>
  <tabstop>proxyGroup</tabstop>
  <tabstop>systemProxyButton</tabstop>
  <tabstop>manualProxyButton</tabstop>
  <tabstop>proxyTypeCombo</tabstop>
  <tabstop>proxyHostEdit</tabstop>
  <tabstop>proxyPortSpin</tabstop>
  <tabstop>proxyUserEdit</tabstop>
  <tabstop>proxyPasswordEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>manualProxyButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
